hnn_core/viz.py

```diff
@@ -1246,6 +1246,15 @@ def plot_laminar_csd(times, data, contact_labels, ax=None, colorbar=True,
     contact_labels : list
         Labels associated with the contacts to plot. Passed as-is to
         :func:`~matplotlib.axes.Axes.set_yticklabels`.
+    vmin, vmax : float, optional
+        data range the colormap covers
+    sink : string color of the sink.
```

Comment:
```suggestion
    sink : str
```

There is a very specific formatting of this docstring ... you should describe the argument below and only mention the datatype here